x-pack/plugins/integration_assistant/server/graphs/ecs/graph.test.ts

```diff
@@ -69,16 +69,22 @@ describe('EcsGraph', () => {
       // When getEcsGraph runs, langgraph compiles the graph it will error if the graph has any issues.
       // Common issues for example detecting a node has no next step, or there is a infinite loop between them.
       try {
-        await getEcsGraph(mockLlm);
+        await getEcsGraph({ model });
       } catch (error) {
-        fail(`getEcsGraph threw an error: ${error}`);
+        throw Error(`getEcsGraph threw an error: ${error}`);
       }
     });
     it('Runs the whole graph, with mocked outputs from the LLM.', async () => {
       // The mocked outputs are specifically crafted to trigger ALL different conditions, allowing us to test the whole graph.
       // This is why we have all the expects ensuring each function was called.
-      const ecsGraph = await getEcsGraph(mockLlm);
-      const response = await ecsGraph.invoke(mockedRequest);
+      const ecsGraph = await getEcsGraph({ model });
+      let response;
+      try {
+        response = await ecsGraph.invoke(mockedRequest);
+      } catch (error) {
+        throw Error(`getEcsGraph threw an error: ${error}`);
```

Comment:
Do we really benefit from catching an error if we immediately rethrow?

Reply:
Tbh this one was a bit weird, because we had some references to "fail" rather than throwing, however jest does not recognise fail anymore, and if I ignore the catch and just //noop, then it sometimes ignores or hides the errors I am looking for (so I end up having to log the error by making a similar change each time), so this time I just kept it in, I can remove it though if it did not make sense.